lib/linux.js

```diff
@@ -77,7 +77,11 @@ const getPathByPid = pid => {
 };
 
 const getPathByPidSync = pid => {
-	return fs.readlinkSync(`/proc/${pid}/exe`);
+	try {
+		return fs.readlinkSync(`/proc/${pid}/exe`);
+	} catch {
+		return null;
```

Comment:
```suggestion
		return;
```

I prefer not using `null`.

Reply:
I see, wasn't sure because of the use of `null` for the title parsing. Fixed with 92b6d381